systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -1,4 +1,4 @@
-function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd)
+function kinsol=doKinematics(model,q,b_compute_second_derivatives,use_mex,qd,options)
```

Comment:
I added an options struct argument to doKinematics since the new doKinematics implementation needs different options. I try to make sure that both passing in the options struct and passing in the booleans work for now, and do not interfere with eachother.